ui/app/templates/components/clients/horizontal-bar-chart.hbs

```diff
@@ -0,0 +1,25 @@
+<svg
+  class="chart is-horizontal"
+  {{on "mouseleave" this.removeTooltip}}
+  {{did-insert this.renderChart @dataset}}
+  {{did-update this.renderChart @dataset}}
```

Comment:
Are the rest of the charts also going to get `did-update` so they're responsive?

Reply:
Maybe. It depends on the data flow. Right now the data is going to come into the dashboard component and be sent to components from there. But this is a good reminder. I'll check-in with Claire on this.

Reply:
Yes - thank you for pointing out! It will be added to the other charts once we have the calendar wired up, which will update the dataset passed in. (Since this code was taken from the bar chart in 1.9 it already has `did-update`. I didn't want to blindly include the helper in the other charts until we had a way to mock the dataset changing and test the functionality)